composer.json

```diff
@@ -15,9 +15,9 @@
     ],
     "require": {
         "php": "^7.4 || ^8.0",
-        "bitbag/shipping-export-plugin": "^1.5",
+        "bitbag/shipping-export-plugin": "^1.5 || ^1.7",
```

Comment:
Why?
`^1.5` => `>= 1.5 && < 2.0` and `^1.7` => `>= 1.7 && < 2.0`? What's the case?

Reply:
Changed to "^1.7", now it should be okay.